include/crm/crm.h

```diff
@@ -123,6 +123,7 @@ extern char *crm_system_name;
 #  define CRM_ATTR_DIGESTS_SECURE   "#digests-secure"
 #  define CRM_ATTR_RA_VERSION       "#ra-version"
 #  define CRM_ATTR_PROTOCOL         "#attrd-protocol"
+#  define CRM_ATTR_FEATURE_SET      "#crm-feature-set"
```

Comment:
Minor request ... let's drop "crm-" and just use "#feature-set". We've been gradually moving away from "crm" as a prefix for things in favor of "pcmk", but in this case any attribute starting with "#" is pacemaker-internal and doesn't really need any prefix beyond that.

Reply:
Dropped the "crm-" prefix as suggested.